great-idea/services.html

```diff
@@ -1,107 +1,134 @@
+          faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button>Learn More</button>
+      </article>
+      <article>
+        <h6>Digital Marketing</h6>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris
+          consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed
+          faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button>Learn More</button>
+      </article>
+      <article>
+        <h6>Web Development</h6>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris
+          consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed
+          faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button>Learn More</button>
+      </article>
+      <article>
+        <h6>iOS Development</h6>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris
+          consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed
+          faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button>Learn More</button>
+      </article>
+      <article>
+        <h6>Android Development</h6>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris
+          consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed
+          faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button>Learn More</button>
+      </article>
+    </section>
+    <section class="facts">
+      <div>
+        <h6>Some Facts About Our Services</h6>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor.
+          Mauris
+          consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed
+          faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <ul>
+          <li>Awesome thing</li>
+          <li>Amazing thing</li>
+          <li>Cool thing</li>
+          <li>Great thing</li>
+        </ul>
+      </div>
+      <img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
+    </section>
+  </main>
+  <footer>
+    <p>Copyright Great Idea! 2018</p>
+  </footer>
 </body>
 
 </html>
```

Comment:
Amazing job on this Shaun, Everything is very organized and well thought out. Excellent work!